server/auth/adfs.ts

```diff
@@ -0,0 +1,98 @@
+import passport from "passport";
+import { IProfile, OIDCStrategy, VerifyCallback } from "passport-azure-ad";
+import express, { NextFunction, Request, Response } from "express";
+
+import "express-session";
+declare module "express-session" {
```

Comment:
looks like type definitions might exist for this under `@types/express-session`?

Reply:
`@types/express-session` is imported, but passport is a little bit outdated and is supposed to make this extension anyway. I'm planning on changing the `any` to something else once i figure out the type of this.